src/bin/p-s.js

```diff
@@ -73,9 +83,30 @@ function onInit() {
     'Check out your scripts in there. Go ahead and update them and add descriptions to the ones that need it'
   ))
   log.info(colors.gray('Your package.json scripts have also been updated. Run `npm start -- --help` for help'))
+  log.info(colors.gray(
```

Comment:
It's possible to use template literals for this multiline string and use [redent](https://github.com/sindresorhus/redent). Might be cleaner but it's personal taste. Adds an extra dependency though.